webhooks_test.go

```diff
@@ -63,3 +65,27 @@ func TestPostWebhook_NotOK(t *testing.T) {
 		t.Errorf("Expected to receive error")
 	}
 }
+
+func TestWebhookMessage_WithBlocks(t *testing.T) {
+	textBlockObject := NewTextBlockObject("plain_text", "text", false, false)
+	sectionBlock := NewSectionBlock(textBlockObject, nil, nil)
+
+	singleBlock := Blocks{BlockSet: []Block{sectionBlock}}
+	twoBlocks := Blocks{BlockSet: []Block{sectionBlock, sectionBlock}}
+
+	msgSingleBlock := WebhookMessage{Blocks: singleBlock}
+	assert.Equal(t, 1, len(msgSingleBlock.Blocks.BlockSet))
+
+	msgJsonSingleBlock, _ := json.Marshal(msgSingleBlock)
+	assert.Equal(t, `{"blocks":[{"type":"section","text":{"type":"plain_text","text":"text"}}]}`, string(msgJsonSingleBlock))
+
+	msgTwoBlocks := WebhookMessage{Blocks: twoBlocks}
+	assert.Equal(t, 2, len(msgTwoBlocks.Blocks.BlockSet))
+
+	msgJsonTwoBlocks, _ := json.Marshal(msgTwoBlocks)
+	assert.Equal(t, `{"blocks":[{"type":"section","text":{"type":"plain_text","text":"text"}},{"type":"section","text":{"type":"plain_text","text":"text"}}]}`, string(msgJsonTwoBlocks))
+
+	msgNoBlocks := WebhookMessage{Text: "foo"}
+	msgJsonNoBlocks, _ := json.Marshal(msgNoBlocks)
+	assert.Equal(t, `{"text":"foo","blocks":null}`, string(msgJsonNoBlocks))
```

Comment:
I think it is better to omit `blocks` properties from encoded json when `blocks` is null.

Reply:
The `blocks` property is now omitted if it is empty. Sadly the travis build crashed due to an error, which based on what I see is not related to the latest commit. Could you please restart the build?